lib/ansible/galaxy/api.py

```diff
@@ -63,25 +66,32 @@ class GalaxyAPI(object):
 
     SUPPORTED_VERSIONS = ['v1']
 
-    def __init__(self, galaxy, name, url, username=None, password=None, token=None):
+    def __init__(self, galaxy, name, url, username=None, password=None, token=None, token_type=None):
         self.galaxy = galaxy
         self.name = name
         self.username = username
         self.password = password
         self.token = token
+        self.token_type = token_type or 'Token'
```

Comment:
It seems like this should potentially be based on the API version that is being used and not something explicitly set. Would there be a reason why this should be configurable outside of using `Bearer` for `v3` and `Token` for older versions?